UI/Resources/zh-Hans.lproj/Localizable.strings

```diff
@@ -0,0 +1,50 @@
+/* Delimiter between lines in an address when displayed inline */
```

Comment:
I removed them temporarily while cleaning up but I think we should keep the translations in here.